edudoc/disciplinary-crash-course/disciplinary-crash-course.md

```diff
@@ -0,0 +1,98 @@
+# Guidelines for Disciplinary Incidents
```

Comment:
I see you renamed the document! I like this new title, and we should also update the filename and containing folder to reflect this new file name: `edudoc/guidelines-for-disciplinary-incidents/guidelines-for-disciplinary-incidents.md`